ddtrace/tracer/tracer.go

```diff
@@ -275,6 +279,12 @@ func (t *tracer) worker(tick <-chan time.Time) {
 	for {
 		select {
 		case trace := <-t.out:
```

Comment:
Will also need to add the processor in `case <-t.stop` below (to cover for the case where `case <-t.stop` is selected before `trace := <-t.out` but there are still traces left to add). The end-to-end test I'm working on allowed me to catch this.